applications/serial_lte_modem/src/slm_settings.c

```diff
@@ -23,6 +23,8 @@ enum fota_stage fota_stage = FOTA_STAGE_INIT;		/* FOTA: stage of FOTA process */
 enum fota_status fota_status;		/* FOTA: OK/Error status */
 int32_t fota_info;		/* FOTA: failure cause in case of error or download percentage*/
 
+int32_t auto_connect = 1;	/* Auto connect network at startup or after FOTA update */
```

Comment:
Not sure whether this should be enabled by default, but I have no preference myself

Reply:
I think it'd better be YES for LwM2M_Carrier and NO for other cases

Reply:
It's safe to be YES here as the real connect logic is enclosed by #if defined(CONFIG_SLM_CARRIER).
There will be no impact on normal SLM build.

Reply:
Yeah I suppose I was referring to the Carrier build specifically, as this will change the default behaviour between versions, but yeah no strong opinion from my side, I will probably always have auto-connect enabled :)